packages/create-discord-bot/src/helpers/packageManager.ts

```diff
@@ -47,19 +47,52 @@ export function resolvePackageManager(): PackageManager {
  * @param packageManager - The package manager to use
  */
 export function install(packageManager: PackageManager) {
-	let installCommand;
+	let installCommand: string[] | string = `${packageManager} install`;
+
+	console.log(`Installing dependencies with ${packageManager}...`);
 
 	switch (packageManager) {
-		case 'npm':
+		case 'yarn':
+			console.log();
+			installCommand = [
+				`${packageManager} set version stable`,
+				`${packageManager} config set nodeLinker node-modules`,
+				`${packageManager} config set logFilters --json '[{ "code": "YN0002", "level": "discard" }, { "code": "YN0013", "level": "discard" }, { "code": "YN0032", "level": "discard" }, { "code": "YN0060", "level": "discard" }]'`,
+				`${packageManager} plugin import interactive-tools`,
+				`${packageManager} plugin import workspace-tools`,
```

Comment:
```suggestion
				`yarn set version stable`,
				`yarn config set nodeLinker node-modules`,
				`yarn config set logFilters --json '[{ "code": "YN0002", "level": "discard" }, { "code": "YN0013", "level": "discard" }, { "code": "YN0032", "level": "discard" }, { "code": "YN0060", "level": "discard" }]'`,
				`yarn plugin import interactive-tools`,
				`yarn plugin import workspace-tools`,
```

just cleaner code

Reply:
I don't think it matters too much does it, the packageManager here cannot be anything different anyway